examples/task-tracker/client/tasks/add_comment_to_task_parameters.go

```diff
@@ -112,6 +133,16 @@ func (o *AddCommentToTaskParams) SetID(id int64) {
 	o.ID = id
 }
 
+// Validate these params
+func (o *AddCommentToTaskParams) Validate(formats strfmt.Registry) error {
+
+	if err := o.Body.Validate(formats); err != nil {
```

Comment:
@casualjim this fails to compile, as that body model has not defined the Validate functions. This is however defined for the Body called here: https://github.com/go-swagger/go-swagger/pull/850/files#diff-ab37f2908b4671ebce8d3bc51539d622R123 ; I'm a bit stuck on this one, could you point me to the correct template location on how I could ensure that body model has the validation defined? Or is it because this model is automatically appended in the generator code (via the name+Body stuff)? If i'm doing crazy stuff in this commit you can also point it out though